.github/CODEOWNERS

```
@@ -1,6 +1,9 @@
 # NHS Notify Code Owners
 
-* @rossbugginsnhs @m-houston @edmundcraske2-nhs @timireland
+* @rossbugginsnhs @m-houston @aidenvaines-bjss @timireland
```

```suggestion
* @NHSDigital/nhs-notify-platform
```
I'd be tempted to make the default code owner platform too, unless you want to create a separate group for the DNS repo?